packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -546,6 +546,8 @@ export class Range {
             return true;
         }
         return isObject<theia.Range>(arg)
+            && isObject(arg.start)
```

Comment:
Why not fix `Position.is()` to accept `unknown` instead?

Reply:
Because I didn't look at `Position.is` and only fixed the new failure here. Do you want to change `Position.is`?

Reply:
If you meant to update `Position.isPosition` then this looks like copied code and I'm not changing copied code.

Reply:
Yes, I mean `isPosition`. It fixes the problem and it's the better fix. What do you mean by "copied code"?

Reply:
It's code copied from VS Code: https://github.com/microsoft/vscode/blob/60f2446458c6806be54eb1d1c877bc42b38a1cd6/src/vs/workbench/api/node/extHostTypes.ts#L68-L80

Reply:
Updated `Position.isPosition` because I don't understand why they changed the param type when copying.